ggml/src/ggml-sycl/common.cpp

```diff
@@ -66,17 +63,21 @@ int64_t downsample_sycl_global_range(int64_t accumulate_block_num, int64_t block
 void ggml_sycl_op_flatten(ggml_backend_sycl_context & ctx, const ggml_tensor *src0,
                                  const ggml_tensor *src1, ggml_tensor *dst,
                                  const ggml_sycl_op_flatten_t op) try {
-    const int64_t nrows0 = ggml_nrows(src0);
+
+    // TODO: What's the use of these?
```

Comment:
I believe that all the unused variables you commented out can be removed without the need of a `TODO`.